src/ensembl/src/content/home/Home.scss

```diff
@@ -1,44 +1,223 @@
 @import 'src/styles/common';
 
 .home {
-  padding-bottom: 80px;
+  display: grid;
+  grid-template-rows: minmax(550px, auto) 85px;
+}
 
-  h2 {
-    color: $grey;
-    font-size: 14px;
-    font-weight: $normal;
-  }
+.container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  background-color: $blue;
```

Comment:
why?

Reply:
Removed. I was just playing with the background color before and forgot to remove it.